api/integrations/fifabot/commands/addMatch.js

```diff
@@ -0,0 +1,114 @@
+const request = require('request-promise');
+const { FIFABOT_API_URL } = require('../variables');
+
+// adicionar partida @player1 @player2 team1score x team2score @player3 @player4
```

Comment:
comments in english pls

Reply:
is not a comment, is the description of the regex